console/backend/src/main/java/org/frankframework/management/web/ServerStatistics.java

```diff
@@ -52,18 +47,6 @@ public ResponseEntity<?> getServerConfiguration() {
 	@PermitAll
 	@GetMapping(value = "/server/health", produces = MediaType.APPLICATION_JSON_VALUE)
 	public ResponseEntity<?> getFrankHealth() {
-		try {
-			return callSyncGateway(RequestMessageBuilder.create(BusTopic.HEALTH));
-		} catch (ApiException e) {
-			Map<String, Object> response = new HashMap<>();
-			response.put("status", HttpStatus.INTERNAL_SERVER_ERROR);
-			response.put("error", "unable to connect to backend system: " + e.getMessage());
-			return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(response);
-		} catch (MessageHandlingException e) {
-			throw e; //Spring gateway exchange exceptions are handled by the SpringBusExceptionHandler
-		} catch (Exception e) {
-			throw new ApiException(e);
-		}
+		return callSyncGateway(RequestMessageBuilder.create(BusTopic.HEALTH));
```

Comment:
Dit was de enige plek met try/catch toestanden die we verder niet gebruiken. Eventuele fouten worden al afgehandeld en een ApiException resulteert al in een internal server error.